lib/cipher.js

```diff
@@ -1,5 +1,7 @@
 const crypto = require('crypto');
 
+const UDP_KEY = crypto.createHash('md5').update('yGAdlopoPVldABfn', 'utf8').digest();
```

Comment:
Let's abstract the key out of the logic; maybe put it in a file called `config.json` or something?

Reply:
Sure no problem